src/main/scala/io/citrine/lolo/bags/MultiTaskBagger.scala

```diff
@@ -105,10 +107,12 @@ class MultiTaskBaggedTrainingResult(
                                      trainingData: Seq[(Vector[Any], Seq[Any])],
                                      useJackknife: Boolean,
                                      biasModels: Seq[Option[Model[PredictionResult[Double]]]],
-                                     rescaleRatios: Seq[Double]
+                                     rescaleRatios: Seq[Double],
+                                     trainingLabels: Seq[Seq[Any]],
+                                     trainingWeights: Seq[Double]
```

Comment:
`trainingLabels` and `trainingWeights` are being piped through so that the correlation over training data can be computed, but that method does not work very well and these arguments will almost certainly be removed by the time this code is released.